File: engine/engine-model/src/main/scala/tools/packet/parcel/DueyParcelReceived.scala
package tools.packet.parcel

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class DueyParcelReceived(private var _from: String, private var _quick: Boolean) extends PacketInput {
  def from: String = _from

  def quick: Boolean = _quick

  override def opcode(): SendOpcode = SendOpcode.PARCEL
}

File: engine/engine-model/src/main/java/tools/packet/parcel/DueyParcelReceived.java
package tools.packet.parcel;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record DueyParcelReceived(String from, Boolean quick) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.PARCEL;
   }
}